code/modules/jobs/job_types/medical.dm

```diff
@@ -194,7 +194,7 @@ Virologist
 	belt = /obj/item/pda/viro
 	ears = /obj/item/radio/headset/headset_med
 	uniform = /obj/item/clothing/under/rank/virologist
-	mask = /obj/item/clothing/mask/surgical
+	backpack_contents = list(/obj/item/clothing/mask/surgical=1) //sunset vox mask

```

Comment:
You can override this one var in `sunset/code/modules/jobs/job_types/medical.dm`